app/src/main/kotlin/com/wire/android/workmanager/worker/NotificationFetchWorker.kt

```diff
@@ -26,13 +27,21 @@ class NotificationFetchWorker
 ) : CoroutineWorker(appContext, workerParams) {
     companion object {
         const val USER_ID_INPUT_DATA = "worker_user_id_input_data"
+        const val TAG = "notification_fetch_worker"
     }
 
     override suspend fun doWork(): Result {
-        inputData.getString(USER_ID_INPUT_DATA)?.let { userId ->
-            wireNotificationManager.fetchAndShowNotificationsOnce(userId)
+        try {
+            inputData.getString(USER_ID_INPUT_DATA)?.let { userId ->
```

Comment:
If we don't have the `USER_ID_INPUT_DATA` then `fetchAndShowNotificationsOnce` won't be executed and we won't catch any exception, this worker will be finished successfully - is it intended? Shouldn't we treat it as an error?